lighthouse-core/config/config.js

```diff
@@ -175,6 +177,31 @@ function validateCategories(categories, audits, auditResults, groups) {
         throw new Error(`${audit.id} references unknown group ${audit.group}`);
       }
     });
+
+    // Validate certification (if any).
+    if (category.certification) {
```

Comment:
do we want to validate any certification in the config, even if a category doesn't call it out?